geodocker-geoserver/Makefile

```diff
@@ -2,36 +2,31 @@ BASE := $(subst -, ,$(notdir ${CURDIR}))
 ORG  := geomesa
 REPO := $(word 2, ${BASE})
 IMG  := quay.io/${ORG}/${REPO}
-GEOMESA_DIST_TARBALL := geomesa-accumulo-dist_2.11-${GEOMESA_VERSION}-bin.tar.gz
-GEOMESA_ACCUMULO_GS_INSTALL := geomesa-accumulo-gs-plugin_2.11-${GEOMESA_VERSION}-install.tar.gz
+GEOMESA_VERSION := 2.0.0-m.1
+ACCUMULO_VERSION := 1.7.3
```

Comment:
use maybe 1.8.1 and 0.93 thrift by default?

Reply:
I made the ACCUMULO version and THRIFT version consistent with the default version found in [geodocker-geomesa/build.sh](https://github.com/geodocker/geodocker-geomesa/blob/18177fd402230bfaabd83c14b3d05cbb33846fa6/build.sh#L5).

If we want to default ACCUMULO_VERSION to 1.8.1, I think it would be best to do that in a separate PR and across this entire project

Reply:
sounds good. I'd recommend that due to issues with 1.7.3 (1.7.4 just came out and might have helped). but just go to 1.8.1 i'd recommend.